app/services/key_rotator/attribute_encryption.rb

```diff
@@ -20,20 +20,13 @@ def uak
     end
 
     def encrypted_attributes
-      encrypted_email_attributes.merge!(other_encrypted_attributes)
-    end
-
-    def encrypted_email_attributes
-      email = EncryptedAttribute.new_from_decrypted(user.email, uak)
-      { encrypted_email: email.encrypted, attribute_cost: new_cost }
-    end
-
-    def other_encrypted_attributes
-      User::ENCRYPTED_ATTRIBUTES.each_with_object({}) do |attribute, result|
-        plain_attribute = user.public_send(attribute)
+      User.attribute_names.grep(/^encrypted_/).each_with_object({}) do |attribute, result|
```

Comment:
rather than sniffing attribute names, what if we had the encryptable module keep track of these when they're added, and add a new method like `encryptable_attributes`, ex:

```ruby
cattr_accessor :encrypted_attributes do
  []
end

def encrypted_attribute(attr_name)
  self.encrypted_attributes << attr_name
end
```